types/signable.go

```diff
@@ -1,5 +1,11 @@
 package types
 
+const (
+	// MaxSignatureSize is a maximum allowed signature size for the Heartbeat,
+	// Proposal and Vote.
+	MaxSignatureSize = 64
```

Comment:
I think we need this to be the `math.Max` of values read from the crypto package incase we introduce bigger signatures